src/py/flwr/server/compat/driver_client_proxy.py

```diff
@@ -16,26 +16,23 @@
 
 
 import time
-from typing import List, Optional
+from typing import Optional
 
 from flwr import common
-from flwr.common import DEFAULT_TTL, MessageType, MessageTypeLegacy, RecordSet
+from flwr.common import DEFAULT_TTL, Message, MessageType, MessageTypeLegacy, RecordSet
 from flwr.common import recordset_compat as compat
-from flwr.common import serde
-from flwr.proto import driver_pb2, node_pb2, task_pb2  # pylint: disable=E0611
+from flwr.proto import task_pb2  # pylint: disable=E0611
```

Comment:
This import shouldn't be necessary

Reply:
True, I moved the function `validate_task_res` that made use of it to the `driver_client_proxy_tests.py` (since we only made use of it in there)